docs/Home.md

```diff
@@ -43,9 +43,18 @@ While the extensions under the Quarkiverse organization are generally independen
 
 # Joining Quarkiverse
 
-The first step is to create an **Extension Proposal** ticket in the [Quarkus issue tracker](https://github.com/quarkusio/quarkus/issues) (make sure to check first if the extension you’re proposing is not created yet) and let us know if you are interested in leading the development of this extension by commenting on the issue. The Quarkus team will review and discuss if the proposal extension should belong to core or live in the Quarkiverse Hub.
+## Getting an extension onboarded:
 
-Projects hosted in the Quarkiverse organization on GitHub should respect the following conventions and policies:
+1. The first step is to create or pick an **Extension Proposal** ticket in the [Quarkus issue tracker](https://github.com/quarkusio/quarkus/issues) and let us know if you are interested in leading the development of this extension by commenting on the issue. The Quarkus team will review and discuss with you and community if the proposal extension should belong to core or live in the Quarkiverse Hub.
```

Comment:
We should probably point out some of the constraints on the extension name which are explained below because it might lead to frustration if people request names that won't be accepted or don't realize that the name they choose will affect other stuff like maven groupId and package names…